frame_capture_extension/inject.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+    const YOUTUBE_IFRAME_ID = '__yt_iframe__';
+    const VID_ANNOT_ROOT_ID = '__vid_annot_root__';
+    const CANVAS_ID = '__image_destination_canvas__';
```

Comment:
If some of these are shared across both the extension and the app, it might be better to keep them in a config file, which is imported by both app and extension.

Reply:
Will look into doing so for chrome. I use firefox usually and module support for extensions is experimental in that.

Reply:
It doesn't have to be modules. Could be environment variables, substituted at build time.